build.gradle

```diff
@@ -25,13 +25,14 @@ buildscript {
                 'androidxPreference': '1.1.1',
                 'materialComponents': '1.1.0',
                 'preferencesFix': '1.1.0',
-                'okio': '2.5.0',
+                'okio': '2.6.0',
+                'desugar_jdk_libs': '1.0.5',
```

Comment:
For consistency can you please change this to use camel case?

Reply:
I've no clue what's the next urgent wish. I recommend to rename it after a merge by yourself, then it's exactly like you want